go-controller/pkg/ovn/ovn_test.go

```diff
@@ -43,11 +44,12 @@ func NewFakeOVN(fexec *ovntest.FakeExec) *FakeOVN {
 	}
 }
 
-func (o *FakeOVN) start(ctx *cli.Context, objects ...runtime.Object) {
+func (o *FakeOVN) start(ctx *cli.Context, egressObjects []runtime.Object, objects ...runtime.Object) {
```

Comment:
It would be cleaner to just have `start()` split `objects` into two arrays by object type. Then you don't have to add a bunch of pointless `nil`s everywhere